File: console.cpp
#include <iostream>
using namespace std;

class console {

void write(int a){
 
 char c = (char)a;
std::cout << c <<std::endl;

}

int main() {
 
   char input;
   cout << "Enter a character: ";
   cin>>input;
  // std::cout << input <<std::endl;
  cout << "ASCII value is: " << int(input);

  // write(35);
    return 0;
}


};

File: JAVA/console.java
import java.util.Scanner;

public class console implements IO{

    @Override
    public char write(int a) {
       char b = (char) a;
       System.out.print(b);
        return b;
    }

    @Override
    public int read() {
        Scanner scan = new Scanner(System.in);
        System.out.print("Enter a character: ");
        char c = scan.next().charAt(0);

        int a = c;
        scan.close();
        return a;
        
    }



    
}
